cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -147,6 +149,14 @@ def check_hosts(small_vm, provider):
         pytest.skip("There is only one host in the provider")
 
 
+@pytest.fixture
+def quota_limit(request, roottenant):
+    in_use_storage = int(float(roottenant.quota["storage"]["in_use"].strip(" GB")))
+    roottenant.set_quota(**{"storage_cb": True, "storage": in_use_storage + 3})
```

Comment:
@valaparthvi 
Why are we type casting **in_use_storage** to ```float``` and again to ```int```. I think, we can remove float typecasting?
```
>>> int(5.1111)
5
>>> int(float(5.1111))
5
>>> 
```

Reply:
because `in_use_storage` is a string, I cannot directly convert string to int.